### ISSUE TITLE
✨ `config.scope`

### ISSUE THREAD

**Issue by 1aron** [master-co/css#132]
### Description

Limit Master CSS syntax to a specific CSS scope:
```ts
export default {
    scope: '#app'
}
```
Your HTML:
```html
<html>
<body id="app">
    <div class="mt:1 text:center"></div>
</body>
</html>
```
Generated CSS:
```css
#app .mt\:1 {
    margin-top: 0.0625rem
}

#app .text\:center {
    text-align: center
}
```

**Comment by 1aron** [comment]
@BenSeage A selector might consist of:
```ts
.map(eachPrefixText => prefixThemeText + scopePrefix + eachPrefixText)
```
https://github.com/master-co/css/blob/beta/packages/css/src/rule.ts#L413

**Comment by 1aron** [comment]
🎉 The feature has been released in [v2.0.0-beta.102](https://github.com/master-co/css/releases/tag/v2.0.0-beta.102)
